code/modules/mob/living/simple_animal/hostile/ordeal/green.dm

```diff
@@ -58,7 +60,18 @@
 				H.apply_damage(20, WHITE_DAMAGE, null, H.run_armor_check(null, WHITE_DAMAGE), spread_damage = TRUE)
 			finishing = FALSE
 
-// Green dawn
+//Green dawn factory spawn
+/mob/living/simple_animal/hostile/ordeal/green_bot/factory
```

Comment:
There is absolutely no reasons to add an entire subtype for it.
Just make factory register a signal for death.